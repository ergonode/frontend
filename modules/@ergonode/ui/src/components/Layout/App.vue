/*
 * Copyright Â© Bold Brand Commerce Sp. z o.o. All rights reserved.
 * See LICENSE for license details.
 */
<template>
    <div class="app">
        <slot />
    </div>
</template>

<script>
export default {
    name: 'App',
};
</script>

<style lang="scss" scoped>
    .app {
        position: relative;
        display: flex;
        height: 100vh;
    }
</style>
