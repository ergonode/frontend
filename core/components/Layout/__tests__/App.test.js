/*
 * Copyright Â© Bold Brand Commerce Sp. z o.o. All rights reserved.
 * See LICENSE for license details.
 */
import { shallowMount } from '@vue/test-utils';
import App from '~/core/components/Layout/App.vue';


describe('Layout/App', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallowMount(App);
    });

    it('Component is named well', () => {
        expect(typeof App.name).toBe('string');
        expect(App.name).toEqual('App');
        expect(wrapper).toMatchSnapshot();
    });
});
