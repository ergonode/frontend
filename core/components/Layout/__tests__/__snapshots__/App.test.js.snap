// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout/App Component is named well 1`] = `<div class="app"></div>`;
